examples/src/main/java/io/dapr/examples/pubsub/http/Subscriber.java

```diff
@@ -17,7 +17,7 @@
  * mvn clean install
  * 2. Run the server:
  * dapr run --app-id subscriber --app-port 3000 --port 3005 -- \
- *   mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.pubsub.http.Subscriber -Dexec.args="-p 3000"
+ *   mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.pubsub.http.Subscriber -D exec.args="-p 3000"

```

Comment:
why do we need space here?

Reply:
It's not required - but at all readmes it's with a space - so my suggestion is to align it

Reply:
I think the other docs are wrong. we need to update the other readmes. I would remove space here.

Reply:
For some reason in the past this command did not work for some on Windows without the space in the past, but the issue doesn't appear to repro now.  This is something we talked about on the other call earlier today.

I think we can remove the space from all docs separately, this PR can keep the spaces.

Reply:
I will clean it up